frontend/src/views/website/runtime/php/create/index.vue

```diff
@@ -165,6 +161,10 @@ import { reactive, ref } from 'vue';
 import Params from '../param/index.vue';
 import EditParams from '../edit/index.vue';
 import DrawerHeader from '@/components/drawer-header/index.vue';
+import { GlobalStore } from '@/store';
+
+const globalStore = GlobalStore();
+const phpDocURL = globalStore.isIntl ? `/user_manual/websites/runtime_php/` : '/user_manual/websites/php/#php_1';
 
 interface OperateProps {
     id?: number;
```

Comment:
The provided code snippet contains some small adjustments. Here's a summary of the changes:

1. **Link URL Update**: The link to the PHP documentation has been updated to use `globalStore.docsUrl`. This allows dynamic retrieval of the correct document URL based on the internationalization setting.

2. **Global Store Access**: A new line is added to import and initialize `GlobalStore`. This setup ensures that you can access global state within your components using `import { globalStore } from '@/store';`.

3. **PHP Documentation Path Adjustment**: A variable `phpDocURL` is defined, which constructs the path to the PHP documentation page based on whether it should be internationalized or not (`/user_manual/websites/runtime_php/` for intl-enabled apps vs. `/user_manual/websites/php/#php_1` otherwise). This approach improves maintainability and scalability by centralizing the logic for determining document paths.

Overall, these changes make the code more flexible and easier to manage if necessary configurations change. It also adheres to good programming practices by separating concerns and encapsulating functionality in separate files like `App.vue`, improving readability, and maintaining consistency across multiple parts of your application.